### ISSUE TITLE
Line chart with multiple Y-axis with common X-axis in center

### ISSUE THREAD

**Issue by Neha3011** [rrag/react-stockcharts#661]
Hi, I am trying to make something like this
<img width="1002" alt="screen shot 2018-09-28 at 6 42 10 pm" src="https://user-images.githubusercontent.com/10024090/46210450-62f6b180-c34e-11e8-917d-569a80d69447.png">

 I have been trying some trial and error since last few hours using some ready to use examples and I am unable to do so. Can you help me understand if it is possible to achieve with this library?

Thanks for your help.

**Comment by dandotevans** [comment]
Yes, this can be done by having 2 charts the 2nd charts origin offset by the first's height. the first would show the xAxis below or the 2nd could show above. 

If this works you should see an edited example below:

[![Edit react-stockcharts-example2](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/m7q82r1x79)
